src/scripts.js

```diff
@@ -95,6 +100,14 @@ searchBar.addEventListener('keypress', (event) => {
 
 searchBtn.addEventListener('click', searchForRecipes);
 
+modalAddBtn.addEventListener('click', (e) => {
+  updateRecipesFromModal(e)
+});
+
+modalRemoveBtn.addEventListener('click', (e) => {
+  updateRecipesFromModal(e)
+});
```

Comment:
Is it possible to dry these up slightly by rather listening to `modal-recipe-btn` and have the parameter of `updateRecipesFromModal` be `e.target.id`?

Reply:
Yes! Done now